app/src/ui/stashing/stash-diff-viewer.tsx

```diff
@@ -0,0 +1,125 @@
+/* tslint:disable:button-group-order */
```

Comment:
:art: can we add some context here about why we've needed to do this?

Reply:
Tracked in #7273.